homeworks/Yaroslav.Melnyk_Maritodd/homework_2/number.js

```diff
@@ -0,0 +1,31 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+function numbers(num) {
+    for (let i = 1; i <= num; i++) {
+        const str = String(i);
+        if (num > 1000) {
+            console.log('Error!');
+            return;
+        }
+        if (str.length === 3) {
+            if (str[0] === str[2]) {
+                console.log(str);
+            }
+        }
+        if (str.length === 4) {
+            if (str[0] === str[3] && str[1] === str[2]) {
+                console.log(str);
+            }
+        }
+    }
+}
```

Comment:
though it's already late, but still this solution is hardcoded and not scalable

Reply:
> though it's already late, but still this solution is hardcoded and not scalable

But Taras said УВАГА. Хто ще не здав 2гу домашку, то можете здавати її одним з пулл реквестом рази з 3ю

Reply:
yes, but it should be done in a correct way :)

Reply:
> yes, but it should be done in a correct way :)

how to do it in a correct way?

Reply:
> yes, but it should be done in a correct way :)

But my program is work. And i came to Academy to learn it)

Reply:
this will work, but what if your number will be 123321? or even 1234554321? you will keep adding conditions to check the length of your string? it's better to automize this

I will accept this, but pls check out `split()`, `reverse()`, `join()` methods, they are very handy for this task